<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionarAtendidos</class>
 <widget class="QDialog" name="gestionarAtendidos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Pacientes Atendidos</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buscar Paciente Atendido:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="buscarAtendidoTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>238</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buscarAtendidoCmd">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registrarOtroPacienteCmd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regresarCmd">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="listaAtendidosTwidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>671</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Elegir Local:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="elegirLocalCbox"/>
    </item>
   </layout>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>16</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
